internal/backend/sessions/sessioncalls/calls.go

```diff
@@ -75,10 +77,13 @@ func New(l *zap.Logger, config Config, svcs *sessionsvcs.Svcs) Calls {
 	}
 }
 
-func (cs *calls) StartWorkers(ctx context.Context) error {
-	tc := cs.svcs.Temporal.TemporalClient()
-	cs.generalWorker = temporalclient.NewWorker(cs.l.Named("sessionscallsworker"), tc, generalTaskQueueName, cs.config.GeneralWorker)
-	cs.uniqueWorker = temporalclient.NewWorker(cs.l.Named("sessionscallsworker"), tc, uniqueWorkerCallTaskQueueName(), cs.config.UniqueWorker)
+func (cs *calls) StartWorkers(ctx context.Context, temporal client.Client) error {
```

Comment:
Don't use a variable name that obscures an import name

Reply:
renamed